### DISCUSSION TITLE
Removing root_admin and replacing it with Filament Shield

### DISCUSSION THREAD

**Discussion by Poseidon281** [pelican-dev/panel#157] [Ideas]
#37 

With regard to this discussion. Is with perhaps an idea to remove the root_admin (which is now used to give a user complete access to everything) and replace it with [Filament Shield](https://filamentphp.com/plugins/bezhansalleh-shield) .

**Comment by lancepioch** [comment]
Thanks, please follow: https://github.com/pelican-dev/panel/issues/193
